src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_configs/launch_analysis.cljs

```diff
@@ -9,12 +9,37 @@
     [org.broadinstitute.firecloud-ui.common.style :as style]
     [org.broadinstitute.firecloud-ui.common.table :as table]
     [org.broadinstitute.firecloud-ui.endpoints :as endpoints]
+    [org.broadinstitute.firecloud-ui.utils :as utils]
     ))
 
 
 (defn- entity->id [entity]
   {:type (entity "entityType") :name (entity "name")})
 
+;; GAWB-662
+;; show Queued first, then Launching, and collapse all remaining states into Active
+(defn queue-status-remap [queue-status]
```

Comment:
You can (ab)use `or` for this.  `(or thing default)` evaluates to `thing` when it exists, or `default` when it doesn't.